badge_server/test_badges.py

```diff
@@ -824,6 +808,24 @@ def test_github_py2py3_py3_incompatible_fresh_nodeps(self):
         self.assertEqual(json_response['right_text'], 'self incompatible')
 
 
+class TestBadgeImageDependency(TestSuccess):
```

Comment:
I think that you should remove `google-api-core` from `UP_TO_DATE_DEPS`.

I think that this might be more readable:
```
# Add up-to-date dependency information for all CompatibilityResults containing
# a single package.
self.success_data = []
for result in RECENT_SUCCESS_DATA:
  if len(result.packages) == 1:
    self.sucess_data.append(result.with_updated_dependency_info(dep_info))
  else:
    self.sucess_data.append(result)

Reply:
done